tests/20-rdfc-verify.js

```diff
@@ -1,84 +1,183 @@
+  implementationName, keyType
+}) {
+  testContext.test.cell = buildResultCell({
+    name: implementationName,
+    keyType,
+    testTitle: testContext.test.title
+  });
+}
+
+/**
+ * Asserts that an implementation supports a `keyType` required by the test
+ * suite's cryptosuite.
+ *
+ * Implementations may support many key types, but they must
+ * support at least the required key types.
+ *
+ * @param {object} options - The options to use for assertion.
+ * @param {object|null} options.testVector - The test vector object to be
+ * validated for the associated keyType. NULL if the keyType is not supported
+ * by the implementation as specified in `supportedEcdsaKeyTypes`.
+ * @param {string} options.keyType - The type of ECDSA key to check for within
+ * the test vector. This specifies the exact key type that the implementation
+ * is expected to support.
+ */
+function expectImplementationTestVector({testVector, keyType}) {
+  expect(
+    testVector,
+    `Implementation not marked as supporting required "${keyType}"! ` +
+    'Is keyType missing from `supportedEcdsaKeyTypes`?'
+  ).to.exist;
+}
+
+function setupReportableTestSuite(runnerContext, name) {
+  runnerContext.matrix = true;
+  runnerContext.report = true;
+  runnerContext.rowLabel = 'Test Name';
+  runnerContext.columnLabel = name;
+
+  runnerContext.implemented = [];
+}
+
+const SUITE = 'ecdsa-rdfc-2019';
+
 const {
   tags,
   credentials,
-  keyTypes
-} = getSuiteConfig('ecdsa-rdfc-2019');
-// only use implementations with `ecdsa-rdfc-2019` verifiers.
-const {match} = endpoints.filterByTag({
+  keyTypes: requiredSupportedKeyTypes
+} = getSuiteConfig(SUITE);
+
+const {match: targetImplementations} = registeredImplementations.filterByTag({
   tags: [...tags],
   property: 'verifiers'
 });
 
+// Pre-build all the requisite test vectors
+const testVectorsByKeyType = await issueTestData({
+  credential: credentials.verify.document,
+  suite: SUITE,
+  keyTypes: requiredSupportedKeyTypes
+});
+
 describe('ecdsa-rdfc-2019 (verify)', function() {
   describe('ecdsa-rdfc-2019 (verifiers)', function() {
-    // this will tell the report
-    // to make an interop matrix with this suite
-    this.matrix = true;
-    this.report = true;
-    this.rowLabel = 'Test Name';
-    this.columnLabel = 'Verifier';
-    this.implemented = [];
-    let testVectors = new Map();
-    before(async function() {
-      testVectors = await issueTestData({
-        credential: credentials.verify.document,
-        suite: 'ecdsa-rdfc-2019',
-        keyTypes
-      });
-    });
-    for(const [name, {endpoints: verifiers}] of match) {
+    setupReportableTestSuite(this, 'Verifier');
+
+    /**
+     * We build a dynamic test suite based on the implementations that support
+     * the `ecdsa-rdfc-2019` suite.
+     *
+     * If the target implementations do not support a specific test we will
+     * skip the test but still keep it defined so that this is obvious in the
+     * results.
```

Comment:
```suggestion
     * If the target implementations do not support a specific test, we will
     * skip the test but still keep it defined, so that this is obvious in the
     * results.
```